src/shop.js

```diff
@@ -1,5 +1,6 @@
 function BinartaShopjs(checkpoint, deps) {
     var shop = this;
+    var application = deps.application || {profile: function () {}};
```

Comment:
The fallback is necessary (for now) because an update is needed in the angular modules to wire the application dependency. If application is undefined, the code would crash. This fallback can be deleted when the angular modules are updated though.